www-client/surf/metadata.xml

```diff
@@ -1,8 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
-<use>
-<flag name="savedconfig">Without a saved config.h, this package depends on <pkg>net-misc/curl</pkg> and <pkg>x11-terms/st</pkg> for a default download mechanism</flag>
-</use>
+	<maintainer type="person">
+		<email>parazyd@dyne.org</email>
+		<name>Ivan J.</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy maintainers</name>
+	</maintainer>
```

Comment:
Sorry but we won't let you take maintainership of this package before you contribute a while. We see too many people come and dissappear after first commit which causes all kind of bureaucratic work for us.

You can still fix bugs and update this package without being the maintainer for it.

Reply:
While I understand this, I also have to say it's also quite discouraging...

Should I just drop the metadata change then and force push again?

Reply:
Thanks for understanding. Yes please, do force push.

And as I said, you *can* still contribute and we will process all PRs to maintainer-needed packages. Our trust has been broken too many times so we are on the defensive here. If you stick around we'll welcome you with open arms, but so far I can only find 1 commit from you in 2018.